modules/microshift-ingress-controller-config.adoc

```diff
@@ -148,14 +269,20 @@ When configured, this field must contain a valid expression or the {microshift-s
 * When the policy is set to `Respond`, the ingress controller sends an HTTP `400` or `408` response, logs the connection if access logging is enabled, and counts the connection in the appropriate metrics.
 
 * When the policy is set to `Ignore`, the `http-ignore-probes` parameter is added to the `HAproxy` process configuration. After this parameter is added, the ingress controller closes the connection without sending a response, then either logs the connection or incrementing metrics.
-//Q: are the incrementing metrics in this case config'd by the admin, or are they automatic to our config?
+//Q: Does the admin need to configure Prometheus or Grafana to have incrementing metrics?
 
 |`logEmptyRequests`
 |Specifies connections for which no request is received and logged. `Log` and `Ignore` are valid values. Empty requests typically come from load-balancer health probes or preconnects and can often be safely ignored. However, these requests can also be caused by network errors and port scans. Therefore, setting this field to `Ignore` can impede detection or diagnosis of network problems and detecting intrusion attempts. The default value is `Log`.
 
 * Setting this value to `Log` indicates that an event should be logged.
 * Setting this value to `Ignore` sets the `dontlognull` option in the `HAproxy` configuration.
 
+|`httpErrorCodePages`
+|Describes custom error code pages. To use this setting, you must configure the `httpErrorCodePages.name` parameter.
+
+|`httpErrorCodePages.name`
+|Specifies custom error code pages. You can only customize errors for `503` and `404` page codes. To customize error code pages, specify a `ConfigMap` name. The `ConfigMap` object must be in the `openshift-config` namespace and contain keys in the `error-page-<error code>.http` format where `<error code>` is an HTTP status code. Each value in the `ConfigMap` must be the full response, including HTTP headers. The default value of this parameter is null.
```

Comment:
In microshift cluster, there isn't the `openshift-config` namespace, so the `ConfigMap` object must be in the `openshift-ingress` namespace.